sdk/resource/resourcekeys/const.go

```diff
@@ -0,0 +1,66 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
These are duplicates of https://github.com/open-telemetry/opentelemetry-go/blob/master/api/standard/resource.go

The existing ones should be used.